test/test_websocket.py

```diff
@@ -6,7 +6,7 @@
 import asyncio
 import json
 import logging
-import snappy
+import snappy  # pylint: disable=import-error
```

Comment:
Why is this needed?

Reply:
most likely for the same reason the pre-commit hook mypy has `args: [--ignore-missing-imports]` and the origin pre-commit hook pylint had `[--disable=E0401]`, to be honest i did not investigate further why the snappy is not available when the pre-commit hooks are run.